src/app/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,19 @@
+import { Card, ListItem, Typography } from '@mui/material';
+import React from 'react';
+
+type TagType = {
+  id: number;
+  title: string;
+};
+
+type Props = { tag: TagType };
+
+export default (props: Props): JSX.Element => (
```

Comment:
you could destructure props here 
```suggestion
export default ({ tag }: Props): JSX.Element => (
```